install/gce-installation-guide.md

```diff
@@ -0,0 +1,119 @@
+    rhel-7-v20180522                                      rhel-cloud         rhel-7                                        READY
+    sles-11-sp4-v20180523                                 suse-cloud         sles-11                                       READY
+    ubuntu-1604-xenial-v20180522                          ubuntu-os-cloud    ubuntu-1604-lts                               READY
+    ubuntu-1804-bionic-v20180522                          ubuntu-os-cloud    ubuntu-1804-lts                               READY
+
+Each distribution has its own project, and each project may host images for multiple versions of the distribution, typically grouped into families. We recommend selecting images by project and family rather than by name as it ensures any scripts or other automation will always be working with a non-deprecated image including security updates, updates to GCE-specific scripts, etc.
+
+### Creating the Image
+
+The following (substituing your preferred disribution project and image family) will produce an image with nested virtualization enabled in your currently active GCE project:
+
+```bash
+SOURCE_IMAGE_PROJECT=ubuntu-os-cloud
+SOURCE_IMAGE_FAMILY=ubuntu-1804-lts
+IMAGE_NAME=${SOURCE_IMAGE_FAMILY}-nested
+
+gcloud compute images create \
+    --source-image-project $SOURCE_IMAGE_PROJECT \
+    --source-image-family $SOURCE_IMAGE_FAMILY \
+    --licenses=https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx \
+    $IMAGE_NAME
+```
+
+`gcloud` should report that the image was created. You can verify that the image has the nested virtualization license with `gcloud compute images show $IMAGE_NAME`. This should produce output like the following (some fields have been removed for clarity and to redact personal info):
+
+    diskSizeGb: '10'
+    kind: compute#image
+    licenseCodes:
+    - '1002001'
+    - '5926592092274602096'
+    licenses:
+    - https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx
+    - https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/licenses/ubuntu-1804-lts
+    name: ubuntu-1804-lts-nested
+    sourceImage: https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-1804-bionic-v20180522
+    sourceImageId: '3280575157699667619'
+    sourceType: RAW
+    status: READY
+
+The primary criterion of interest here is the presence of the "enable-vmx" license. Without that licence Kata will not work. The presence of that license instructions the Google Compute Engine hypervisor to enable Intel's VT-x instructions in virtual machines created from the image. Note that nested virtualizaiton is only available in VMs running on Intel Haswell or later CPU microarchitectures.
+
+### Verifying VMX is Enabled
+
+Assuming you created a nested-enabled image using the instructions above you can verify that VMs created from this image are VMX-enabled with the following:
```

Comment:
Assuming you created a nested-enabled image using the previous instructions, verify that VMs created from this image are VMX-enabled with the following:

Reply:
Done.